src/Tools/Shared/CommandLine/CommandLineApplicationExtensions.cs

```diff
@@ -26,7 +26,7 @@ public static CommandOption Option(this CommandLineApplication command, string t
         => command.Option(
             template,
             description,
-            template.IndexOf("<", StringComparison.Ordinal) != -1
+            template.Contains("<")
```

Comment:
```suggestion
            template.Contains('<')
```

Reply:
This just re-broke the build... :( I'm reverting.

Reply:
Reverting won't fix it either. This must be used in multiple tfms, one without char based `Contains` method.

Reply:
Reverting your change will fix it.

Reply:
This set of analysis rules is terrible. Am I really supposed to #if here?

Reply:
This is a bad experience.